osmaxx-py/excerptconverter/dummyexcerptconverter/dummy_excerpt_converter.py

```diff
@@ -7,120 +7,116 @@
-    def export_options():
-        return {
-            'detail_level': {
-                'label': 'Detail level',
-                'type': 'choice',
-                'default': 'verbatim',
-                'values': [
-                    {'name': 'verbatim', 'label': 'Verbatim'},
-                    {'name': 'simplified', 'label': 'Simplified'}
-                ]
-            }
-        }
-
-    @staticmethod
-    def create_output_files(execution_configuration, extraction_order, supported_export_formats, converter_helper):
-        for format_key in execution_configuration['formats']:
-            output_file = models.OutputFile.objects.create(
-                mime_type=supported_export_formats[format_key]['mime_type'],
-                extraction_order=extraction_order
-            )
-
-            if not os.path.exists(private_storage.location):
-                os.makedirs(private_storage.location)
-
-            file_name = str(output_file.public_identifier) + '.' + \
-                supported_export_formats[format_key]['file_extension']
-            file_content = ContentFile(str('detail level: ' + execution_configuration['options']['detail_level']))
-            fs_file = private_storage.save(file_name, file_content)
-
-            # file must be committed, so reopen to attach to model
-            output_file.file = fs_file
-            output_file.save()
-
-            if private_storage.exists(file_name):
-                message_level = messages.SUCCESS
-                message_text = _('"%s" created successful' % file_name)
-            else:
-                message_level = messages.ERROR
-                message_text = _('Creation of "%s" failed!' % file_name)
-            converter_helper.inform_user(message_level, message_text, email=False)
-
-    @staticmethod
-    @shared_task
-    def execute_task(extraction_order_id, supported_export_formats, execution_configuration):
-        wait_time = 0
-        # wait for the db to be updated!
-        extraction_order = None
-        while extraction_order is None:
-            try:
-                extraction_order = models.ExtractionOrder.objects.get(pk=extraction_order_id)
-            except models.ExtractionOrder.DoesNotExist:
-                time.sleep(5)
-                wait_time += 5
-                if wait_time > 30:
-                    raise
-
+NAME = 'Dummy'
+EXPORT_FORMATS = {
+    'txt': {
+        'name': 'Text (.txt)',
+        'file_extension': 'txt',
+        'mime_type': 'text/plain'
+    },
+    'markdown': {
+        'name': 'Markdown (.md)',
+        'file_extension': 'md',
+        'mime_type': 'text/markdown'
+    }
+}
+EXPORT_OPTIONS = {
+    'detail_level': {
+        'label': 'Detail level',
+        'type': 'choice',
+        'default': 'verbatim',
+        'values': [
+            {'name': 'verbatim', 'label': 'Verbatim'},
+            {'name': 'simplified', 'label': 'Simplified'}
+        ]
+    }
+}
+
+
+def converter_configuration():
```

Comment:
We can't inherit this function because we are no longer using classes. So it's the best solution to implement the returned data structure ONLY once.
@njordan-hsr Do you have a better idea?